### DISCUSSION TITLE
Drone Mining Confusion

### DISCUSSION THREAD

**Discussion by Kename** [Wilm0rien/EveOnlineMiningYield#7] [General]
Will you be updating the various boats' drone yields?
- Right now the covetor shows different yields, but they should all be same with no bonuses to a particular hull.
- The exhumers also don't get drone mining bonuses. Should they be the same?
- The rigs are different on the barges, so that might make it hard to for people to compare them.
- The covetor only shows 2 rigs.

Those are just the ones off the top of my head.

**Comment by Wilm0rien** [comment]
> Right now the covetor shows different yields, but they should all be same with no bonuses to a particular hull.

the covetor is different because it cannot fit the same rigs as the procurer and retriver due of CPU limitation of this hull

> The exhumers also don't get drone mining bonuses. Should they be the same?

which drone mining bonuses do you mean? the exhumers get the following drone mining bonuses
```perl
	use strict;
	# skills 
	my $Drone_Interfacing_Skill           = 5;
	my $Mining_Drone_Operation_Skill      = 5;
	my $Mining_Drone_Specialization_Skill = 5;
	my $drone_base = 33; # Mining Drone II
	my $drone_yield =$drone_base
		* (1 + 0.1 * $Drone_Interfacing_Skill)
		* (1 + 0.05 * $Mining_Drone_Operation_Skill)
		* (1 + 0.02 * $Mining_Drone_Specialization_Skill)
		* (1 + 0.15 ) # medium mining drone augmentor II
		* (1 + 0.15 ) # medium mining drone augmentor II
		;
	printf("%3.2f", $drone_yield); # prints 90.01
```
this corresponds to the ingame value

![image](https://user-images.githubusercontent.com/20628481/150638675-afceb179-c398-4500-ac76-2e4cbc6c7b50.png)

> The rigs are different on the barges, so that might make it hard to for people to compare them.

the rigs are different because the barges cannot fit the same rigs. however if you want to compare how many yield you can achieve with a particular barg you need to fit the best rigs which fit into that barge. 

> The covetor only shows 2 rigs.

the response above.
